core/Test/Tasty/Ingredients/ConsoleReporter.hs

```diff
@@ -76,9 +76,10 @@ import Data.Foldable (foldMap)
 -- @since 0.12
 data TestOutput
   = PrintTest
-      {- test name         -} String
-      {- print test name   -} (IO ())
-      {- print test result -} (Result -> IO ())
+      {- test name           -} String
+      {- print test name     -} (IO ())
+      {- print test progress -} (Progress -> IO ())
+      {- print test result   -} (Result -> IO ())
```

Comment:
This is a breaking change, please attach `@since 1.5` to the constructor and update changelog.

Reply:
Er, the constructor is already present in 1.4.3, so attaching `@since 1.5` to the constructor is not correct.  It should be attached to the new field.  

Note also that the constructor docs is broken atm, since it gives explanation to 3 of the now 4 fields.

I thought that Haddock allows to document constructor fields separately, so why not turn the vanilla field comments into proper haddock ones?

Reply:
I pushed a fix for this to show what I mean.